client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -63,7 +63,15 @@
         };
 
         this.update = function update(dictionary, data, success, error) {
-            return api.save('dictionaries', dictionary, data).then(success, error);
+            var sendData = {};
+            angular.forEach(data, function(val, key) {
+                if (key !== 'content') {
+                    sendData[key] = val === null? val: val.toString();
+                } else {
+                    sendData[key] = val;
+                }
```

Comment:
Hm, clarity was actually the reason for proposing a change. With `if` you have two different paths where `sendData[key]` can be set to `val`, which is redundant and hence a bit less readable IMO.